hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -1,51 +1,15 @@
 apiVersion: v1
 kind: Template
-metadata:
-  name: olm-artifacts-template
-
 parameters:
 - name: REGISTRY_IMG
   required: true
 - name: CHANNEL
-  value: staging
+  required: true
 - name: IMAGE_TAG
-  value: latest
-
-objects:
```

Comment:
looking at aws olm-artifacts-template I see CatalogSource, Subscription, and OperatorGroup https://github.com/openshift/aws-account-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml